### PR TITLE
Frontend: Standalone Application integration

### PR DESCRIPTION
**Pull request by cannarelladev**
Just a minor update of the Dashboard to integrate the Standalone Application features

### CODE REVIEW

**Comment by cannarelladev** [comment]
Ready for Review @QcFe @giorio94 (there aren't significative modifications, just some minor changes to integrate Standalone Applications, i think is ready to be approved, thx)

**Comment by cannarelladev** [comment]
THIS PR CAN BE MERGED ONLY AFTER THE `Standalone applications` [#716](https://github.com/netgroup-polito/CrownLabs/pull/716)

File: frontend/src/utilsLogic.tsx [hunk 1]
```diff
@@ -151,6 +152,13 @@ export const makeGuiInstance = (
   const { guiEnabled, persistent, environmentType } =
     (environmentList ?? [])[0] ?? {};
 
+  let myDrive = '';
+  if (
+    environmentType === EnvironmentType.Standalone ||
+    environmentType === EnvironmentType.Container
+  ) {
+    myDrive = status?.myDriveUrl ?? '';
+  }
```
Comment (giorio94) [inline frontend/src/utilsLogic.tsx:161]: Is this actually needed? In case the environment is VM-based, the `myDriveUrl` is always empty (same below).

**Review by giorio94 — APPROVED** [review]

File: frontend/src/utilsLogic.tsx (hunk 1 above)
Comment (cannarelladev) [reply frontend/src/utilsLogic.tsx:161]: you're right

File: frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ const getSSHTooltipText = (
   isInstanceReady: boolean,
   environmentType: EnvironmentType
 ) => {
+  if (environmentType === EnvironmentType.Standalone)
+    return 'Standalone Applications does not support SSH connection (yet!)';
```
[the remark this answers is not in the document]
Comment (cannarelladev) [reply frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx:15]: ok

**Pull request merged**